build.sbt

```
@@ -75,6 +75,10 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
       "org.typelevel" %%% "munit-cats-effect-3" % munitCatsEffectV % Test,
     ),
 
+  ).jsSettings(
+    scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule)},
+    scalaJSUseMainModuleInitializer := true,
+    scalaJSStage in Global := FullOptStage,
```

Won't take responsibility for any consequences of this :P

```suggestion
    Compile / scalaJSStage := FullOptStage,
```

---

Just for fun record keeping, we tried this and it brought back `fullLinkJS` in testing